book/website/community-handbook/bookdash/bookdash-application.md

```diff
@@ -0,0 +1,95 @@
+(ch-bookdash-application)=
+# Application and Review Process
+
+The application, review, and selection process have been developed to provide a fair opportunity for all the interested members to show their interest in attending our book dash events.
+
+In this subchapter, we have provided details on the book dash's application and reviewing process.
+
+## Call for application
+
+The book dash event is organised for up to 15 participants.
+The reason behind keeping it small is to allow the core team members to offer specific mentorship to the invited participants and make this an interactive, intimate, fun, and productive event.
+
+To fairly make this selection, a call for application is opened at least 3 months in advance so that interested members can share their skills, interest, and expected outcomes through a short application.
+The applications are kept open for a minimum of 4 weeks and the timeline is communicated clearly through our newsletters, Twitter feed, and application form.
+
+We want to make sure that the selected group of participants brings diverse skills, knowledge, and experience to _The Turing Way_, therefore we request our applicants to answer some demographic questions (on gender, ethnicity, and career stage).
+We also ask our applicants to indicate if they consider themselves to be a member of one or multiple historically underrepresented groups.
+Here they can add any other information about themselves that they think the core team should know when reviewing their application (such as LGBTQ+, immigrant, disabled, parent, carer, POC members).
+
+We seek to remove any barriers that might prevent our participants from attending the event.
+Hence, we use this form to assess what support we can offer, including funding requirements, to our participants in ensuring that they can comfortably attend the event.
+Applicants can mention their travel, accommodation, childcare, and accessibility requirements for their participation.
+
+All these responses are securely handled under the [GDPR (General Data Protection Regulation) law](https://en.wikipedia.org/wiki/General_Data_Protection_Regulation) (learn about creating [GDPR-proof surveys](https://www.surveylegend.com/gdpr/how-to-make-gdpr-proof-surveys-or-forms/)).
+
+## Reviewing process
+
```

Comment:
Different numerical scoring systems referred to, (both 0,1,2 and 1,2,3) - best to pick one for consistency.